vms/volunteer/models.py

```diff
@@ -11,6 +11,7 @@
 
 class Volunteer(models.Model):
     id = models.AutoField(primary_key=True)
+    is_verified = models.BooleanField(default=False)
```

Comment:
We don't need such field, django already gives us `is_active` field which is generally used for this case.

When an User clicks on verify click we set this flag as true and they can signup, etc...